src/DDTrace/OpenTracer/SpanContext.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace DDTrace\OpenTracer;
+
+use DDTrace\Contracts\SpanContext as SpanContextInterface;
+use DDTrace\ID;
+use DDTrace\SpanContext as DDSpanContext;
+use OpenTracing\SpanContext as OTSpanContext;
+
+final class SpanContext implements OTSpanContext
+{
+    /**
+     * @var SpanContextInterface
+     */
+    private $context;
+
+    /**
+     * @param SpanContextInterface $context
+     */
+    public function __construct(SpanContextInterface $context)
+    {
+        $this->context = $context;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getBaggageItem($key)
+    {
+        return $this->context->getBaggageItem($key);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function withBaggageItem($key, $value)
+    {
+        return $this->context->withBaggageItem($key, $value);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getIterator()
+    {
+        return $this->context->getIterator();
+    }
+
+    /**
+     * @return SpanContextInterface
+     */
+    public function unwrapped()
+    {
+        return $this->context;
+    }
+
+    /**
+     * Converts an OpenTracing SpanContext instance to a DD one
+     *
+     * @param OTSpanContext $otContext
+     * @return DDSpanContext
+     */
+    public static function toDDSpanContext(OTSpanContext $otContext)
```

Comment:
As I mentioned above, there may be cases when this "copying" vs "wrapping" may bite us back. Let's just make some more reasoning about it.